resources/virt.go

```diff
@@ -82,6 +99,14 @@ func (obj *VirtRes) Init() error {
 		}
 		libvirtInitialized = true
 	}
+	if u, err := url.Parse(obj.URI); err != nil {
+		log.Printf("%s[%s]: Parsing URI failed: %s", obj.Kind(), obj.GetName(), obj.URI)
+	} else {
```

Comment:
Not really since I don't need u other than for this and u is only visible within the if else portion